articles/how-to-create-a-dataset-for-machine-learning/index.md

```diff
@@ -0,0 +1,108 @@
+
+Using GANs effectively generates synthetic videos and images that look realistic for use in different applications. It takes in existing data and creates new data that looks like your original dataset. Thus, generating more data.
+
+#### Data Augmentation
+
+Data augmentation is another method for data acquisition. The process involves augmenting existing datasets with newly-acquired external data. Some basic steps in the data augmentation process might include cropping, flipping, rotating, brightness and contrast changing of input images.
+
+This technique enhances the size and quality of training datasets enabling you to collect more data without actually going out to physically collect more data. Another advantage of data augmentation is that it makes a models generalize better to newly unseen data.    
+
+### Data Cleaning
+
+If you do have enough data, but either the quality of the dataset isn't that great (e.g., data is noisy), or there's an issue with the general formatting in your dataset (e.g., some data intervals are in minutes while some in hours), we now move on to the second process which involves cleaning the data.
+
+You can perform data operations manually, but it is labor-intensive and would take a lot of time. Alternatively, you can leverage already built systems and frameworks to help you achieve the same goal easier and faster.
+
+They include:
+
+1. [HoloClean](http://www.holoclean.io/) 
+
+HoloClean repairs, cleans and enriches data. The system leverages value correlations, quality rules, and reference data to help build probabilistic models that capture the process of data generation. It also helps data scientists save on the time needed to clean data.
+
+2. [ActiveClean](https://activeclean.github.io/) 
+
+ActiveClean is an iterative cleaning framework that cleans samples of data based on how much cleaning improves the models' accuracy. This means that you only need to clean a small subset of the data to achieve a model similar to a fully cleaned dataset.
+
+3. [BoostClean](https://arxiv.org/pdf/1711.01299.pdf) 
+
+BoostClean automatically detects and repairs errors in data using statistical boosting. Statistical boosting enables the system to find the best ensemble of pairs that maximize the final model's accuracy.
+
+4. [MLClean](https://arxiv.org/pdf/1904.10761.pdf)
+
+MLClean is the most recent data cleaning framework. The framework performs three main tasks: data sanitization, traditional data cleaning, and unfairness mitigation in data. It also integrates artificial intelligence to clean data to achieve robust, accurate, and fair models.     
```

Comment:
```suggestion
MLClean is the most recent data cleaning framework. The framework performs three main tasks: 
- Data Sanitization
- Traditional data cleaning
- Unfairness mitigation in data 
It also uses Artificial Intelligence to clean data to achieve robust, accurate, and fair models. 
```
A few suggestions:
- Give a single line description for each of the tasks.
- `It also integrates artificial intelligence`. This statement is more generalized. Since your article is also related to AI, the reader would want to know how it is done.